src/App.js

```diff
@@ -2,77 +2,50 @@ import React, { Component } from 'react';
 import './App.css';
 
 class App extends Component {
-  _isMounted = false;
-
   state = {
-    source: 'https://i.imgur.com/VYUdUNJ.png',
```

Comment:
Since you are calling fetchCat() in componentDidMount, this will be overwritten, so there is not point in initializing state to this link. Rather initialize it to empty string or null